src/redux/actions/concepts/dictionaryConcepts.js

```diff
@@ -172,6 +165,52 @@ export const filterByClass = (
   dispatch(fetchDictionaryConcepts(conceptType, conceptOwner, conceptName, query));
 };
 
+export const queryPossibleAnswers = query => async (dispatch) => {
+  const url = `concepts/?q=${query}&limit=0`;
+  try {
+    const response = await instance.get(url);
+    dispatch({
+      type: QUERY_POSSIBLE_ANSWER_CONCEPTS,
+      payload: response.data,
+    });
+  } catch (error) {
+    notify.show('There is something wrong with your connection', 'error', 3000);
+  }
+};
+
+export const addSelectedAnswersToState = answers => (dispatch) => {
+  dispatch({ type: ADD_SELECTED_ANSWERS, payload: answers });
+};
+
+export const changeSelectedAnswer = data => (dispatch) => {
+  dispatch({ type: CHANGE_ANSWER_MAPPING, payload: data });
+};
+
+export const addAnswerMappingToConcept = async (url, source, answers) => {
+  const user = localStorage.getItem('username');
+  const mappingUrl = `/users/${user}/sources/${source}/mappings/`;
+  const answerMappings = answers.map(answer => (answer.map_scope === 'Internal' ? {
```

Comment:
Don't we already have a constant for that text?

Reply:
You can always create it if it does not exist.

Reply:
@dkayiwa let me create it.